UnitTests/CommonTestUtils/SubmoduleTestHelpers.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Threading;
+using GitCommands;
+using GitCommands.Submodules;
+
+namespace CommonTestUtils
+{
+    public class SubmoduleTestHelpers
+    {
+        public static SubmoduleInfoResult UpdateSubmoduleStatusAndWaitForResult(ISubmoduleStatusProvider provider, GitModule module, bool updateStatus = false)
```

Comment:
Is there a compelling reason to put it here? It feels like an implementation detail for `FormBrowseLeftPanelSubmodulesTests`

Reply:
I factored this out of SubmoduleStatusProviderTests so that I could now use it in both places.